### PR TITLE
Implement boostrap UI menu option and dialog

### PR DESCRIPTION
**Pull request by effectsToCause**

### CODE REVIEW

File: src/Makefile.qt.include [hunk 1]
```diff
@@ -27,7 +27,8 @@ QT_FORMS_UI = \
   qt/forms/sendcoinsdialog.ui \
   qt/forms/sendcoinsentry.ui \
   qt/forms/signverifymessagedialog.ui \
-  qt/forms/transactiondescdialog.ui
```
Comment (mderasse) [inline src/Makefile.qt.include:30]: Alphabetical order please

File: src/Makefile.qt.include [hunk 2]
```diff
@@ -79,7 +80,8 @@ QT_MOC_CPP = \
   qt/moc_walletcontroller.cpp \
   qt/moc_walletframe.cpp \
   qt/moc_walletmodel.cpp \
-  qt/moc_walletview.cpp
+  qt/moc_walletview.cpp \
+  qt/moc_bootstrapdlg.cpp
```
Comment (mderasse) [inline src/Makefile.qt.include:84]: Alphabetical order please

File: src/Makefile.qt.include [hunk 3]
```diff
@@ -165,7 +167,8 @@ BITCOIN_QT_H = \
   qt/walletmodel.h \
   qt/walletmodeltransaction.h \
   qt/walletview.h \
-  qt/winshutdownmonitor.h
+  qt/winshutdownmonitor.h \
+  qt/bootstrapdlg.h
```
Comment (mderasse) [inline src/Makefile.qt.include:171]: Alphabetical order please

File: src/Makefile.qt.include [hunk 4]
```diff
@@ -247,7 +250,8 @@ BITCOIN_QT_BASE_CPP = \
   qt/rpcconsole.cpp \
   qt/splashscreen.cpp \
   qt/trafficgraphwidget.cpp \
-  qt/utilitydialog.cpp
+  qt/utilitydialog.cpp \
+  qt/bootstrapdlg.cpp
```
Comment (mderasse) [inline src/Makefile.qt.include:254]: Alphabetical order please

File: src/qt/bitcoingui.cpp [hunk 5]
```diff
@@ -962,6 +968,13 @@ void BitcoinGUI::showHelpMessageClicked()
     helpMessageDialog->show();
 }
 
+void BitcoinGUI::bootstrapClicked()
+{
+    auto bootdlg = new Bootstrapdlg(this);
```
Comment (mderasse) [inline src/qt/bitcoingui.cpp:973]: Can we call that dialog like the others ?

Comment (effectsToCause) [reply src/qt/bitcoingui.cpp:973]: fixing will resubmit pull request

**Pull request closed without merging**